cli/crates/backend/build.rs

```diff
@@ -1,4 +1,5 @@
 fn main() {
+    println!("cargo:rerun-if-changed=src/api/graphql/api.graphql");
```

Comment:
The function call below already does this btw: https://github.com/obmarg/cynic/blob/main/cynic-codegen/src/registration.rs#L194-L196

Reply:
I did that by reflex, if it's automatic the line can be removed, good catch!

Reply:
Done